### DISCUSSION TITLE
Client.create_tweet() works client.get_users gives 401

### DISCUSSION THREAD

**Discussion by adaki2004** [tweepy/tweepy#2042] [Questions] [answered]
Hey There,

I do not exactly get why I can create a tweet (create_tweet) and why getting 401 with get_users()
(OAuth 2.0 Client ID and Client Secret)

```
import tweepy
# Twitter API credentials
consumer_key = "XXX"
consumer_secret = "XXX"
access_token = "XXX"
access_token_secret = "XXX"
bearer_token="XXX"

client = tweepy.Client(bearer_token=bearer_token)

client = tweepy.Client(
    consumer_key=consumer_key, consumer_secret=consumer_secret,
    access_token=access_token, access_token_secret=access_token_secret
)

client.create_tweet(text="GM")

user_ids = [2244994945, 6253282]

# By default, only the ID, name, and username fields of each user will be
# returned
# Additional fields can be retrieved using the user_fields parameter
response = client.get_users(ids=user_ids)
```

Please let me know what am i doing wrong !

**Comment by Harmon758** [comment, marked as answer]
You're using OAuth 1.0a, not OAuth 2.0, as you're re-defining `client` as the second instance of `Client`.

By default, [`Client.get_users`](https://tweepy.readthedocs.io/en/v4.12.1/client.html#tweepy.Client.get_users) does not use OAuth 1.0a User Context to authenticate.
You need to set its `user_auth` parameter to `True`.
